hugr-model/src/v0/text/parse.rs

```diff
@@ -170,21 +171,23 @@ impl<'a> ParseContext<'a> {
             }
 
             Rule::term_ext_set => {
-                let mut extensions = Vec::new();
-                let mut rest = None;
+                let mut items = BumpVec::with_capacity_in(inner.len(), self.bump);
```

Comment:
Another place where I wonder if `BumpVec::from_iter_in` would do the job?

Reply:
`BumpVec::from_iter_in` does not support propagating errors via `Result`s.